src/components/RequestCard/index.tsx

```diff
@@ -100,52 +104,64 @@ const RequestCard: React.FC<RequestCardProps> = ({ request, onTitleData }) => {
       }}
     >
       <div className="flex flex-col flex-1 min-w-0 pr-4">
-        <h2 className="overflow-hidden text-base text-white cursor-pointer sm:text-lg overflow-ellipsis whitespace-nowrap hover:underline">
-          <Link
-            href={request.type === 'movie' ? '/movie/[movieId]' : '/tv/[tvId]'}
-            as={
-              request.type === 'movie'
-                ? `/movie/${request.media.tmdbId}`
-                : `/tv/${request.media.tmdbId}`
-            }
-          >
+        <Link
+          href={
+            request.type === 'movie'
+              ? `/movie/${requestData.media.tmdbId}`
+              : `/tv/${requestData.media.tmdbId}`
+          }
+        >
+          <a className="overflow-hidden text-base text-white cursor-pointer sm:text-lg overflow-ellipsis whitespace-nowrap hover:underline">
             {isMovie(title) ? title.title : title.name}
-          </Link>
-        </h2>
-        <Link href={`/users/${requestData.requestedBy.id}`}>
-          <a className="flex items-center group">
-            <img
-              src={requestData.requestedBy.avatar}
-              alt=""
-              className="w-4 mr-1 rounded-full sm:mr-2 sm:w-5"
-            />
-            <span className="text-xs truncate sm:text-sm group-hover:underline">
-              {requestData.requestedBy.displayName}
-            </span>
           </a>
         </Link>
-        {requestData.media.status && (
-          <div className="mt-1 sm:mt-2">
+        <div className="card-field">
+          <Link href={`/users/${requestData.requestedBy.id}`}>
+            <a className="flex items-center group">
+              <img
+                src={requestData.requestedBy.avatar}
+                alt={requestData.requestedBy.displayName}
+                className="avatar-sm"
+              />
+              <span className="truncate group-hover:underline">
+                {requestData.requestedBy.displayName}
+              </span>
+            </a>
+          </Link>
+        </div>
+        <div className="card-field">
+          <span className="card-field-name">
+            {intl.formatMessage(messages.status)}
+          </span>
+          {requestData.media[requestData.is4k ? 'status4k' : 'status'] ===
+            MediaStatus.UNKNOWN ||
+          requestData.status === MediaRequestStatus.DECLINED ? (
+            <Badge badgeType="danger">
+              {requestData.status === MediaRequestStatus.DECLINED
+                ? intl.formatMessage(globalMessages.declined)
+                : intl.formatMessage(globalMessages.failed)}
+            </Badge>
+          ) : (
             <StatusBadge
               status={
-                requestData.is4k
-                  ? requestData.media.status4k
-                  : requestData.media.status
+                requestData.media[requestData.is4k ? 'status4k' : 'status']
```

Comment:
I don't understand why this was changed?

Reply:
I realized that the badge was not being displayed for failed requests, so I copied the status badge code/logic that we have in the request list.  I think it's best to be consistent?  And I prefer this since it's more readable/compact.

Reply:
More compact does not necessarily mean more readable. I prefer the previous style, but it's subjective. You also lose auto-completion writing it this way.

Reply:
That's true.  I'll update it in both locations then?

Reply:
I was previously under the impression that this way was preferable since we deal with the download status in the same way (but I suppose that is also more complex since it's an array and not as simple to read as this case).

Reply:
It's useful for inline conditional values, but this is just using it as a simple ternary statement. I don't really think it's super important. If you do change it, we should update the RequestItem also. (I copied the code from the old component, which is why it's unchanged)

I think the main thing about this is we are changing it here just to change it. There really isn't a point other than "I like it the other way?" You can argue consistency but code is organic and changes all the time. We can't maintain a specific and consistent style across the code base and we would be crazy to try to. We've already spent too much time over something very insignificant :D

Reply:
@sct So I actually did copy this from `RequestItem`.  I only changed it because I noticed they were different but the badges should really be the same.  Whichever way you want to keep it, I do prefer that it's the same in both places!  That was my original intention.

Reply:
You will notice things are different all the time, but sometimes we gotta say, "it doesn't matter, it works!" We have tons of contributors, coding styles, preferences. Adding extra changes to a PR to keep things consistent (if that's not the PR's purpose) increases the cognitive load of the PR with no real gain.

I think it's best to leave it for now, since it's the same, but let's keep this in mind in the future.

Reply:
You got it, boss!